doc/topics/the-event-loop-timers-and-nexttick.md

```diff
@@ -69,7 +69,7 @@ actually uses - are those above._
 
 * **timers**: this phase executes callbacks scheduled by `setTimeout()`
  and `setInterval()`.
-* **I/O callbacks**: most types of callback except timers, `setImmedate()`, close
+* **I/O callbacks**: most types of callback except timers, `setImmediate()`, close
```

Comment:
I agree, reading the full list one can infer something like this:

"executes almost all callbacks with the exception of close callbacks, the ones scheduled by timers and `setImmediate()`"

It is a long sentence but I think it's better to be explicit.